extensions/elytron-security/runtime/src/main/java/io/quarkus/elytron/security/runtime/DefaultRoleDecoder.java

```diff
@@ -21,7 +22,8 @@
  */
 public class DefaultRoleDecoder {
 
-    private static final String DEFAULT_ATTRIBUTE_NAME = "groups";
+    @ConfigProperty(name = "quarkus.security.default-role-decoder-name", defaultValue="groups")
```

Comment:
It might make sense to make this property name more neutral I guess, so that the custom mappers can also rely on it.
How about `quarkus.security.roles-claim-name`  ?

Reply:
Thanks for your feedback, I changed it now to `quarkus.security.roles-claim-name`